tgstation.dme

```diff
@@ -1150,6 +1150,7 @@
 #include "code\modules\client\preferences.dm"
 #include "code\modules\client\preferences_savefile.dm"
 #include "code\modules\client\preferences_toggles.dm"
+#include "code\modules\client\verbs\etips.dm"

```

Comment:
it would be awesome if you could PR that to me. I've been trying to figure it out but the format is kinda complex.